drivers/hmis/app/models/hmis/hud/enrollment.rb

```diff
@@ -22,6 +22,7 @@ class Hmis::Hud::Enrollment < Hmis::Hud::Base
   belongs_to :project, **hmis_relation(:ProjectID, 'Project'), optional: true
   has_one :exit, **hmis_relation(:EnrollmentID, 'Exit'), dependent: :destroy
   has_many :services, **hmis_relation(:EnrollmentID, 'Service'), dependent: :destroy
+  has_many :custom_services, **hmis_relation(:EnrollmentID, 'CustomService'), dependent: :destroy
```

Comment:
hmmmm I hadn't quite thought this through. I'm not sure if this is going to work for graphql resolution 😒

The GraphQL schema shouldn't make a distinction between HUD services and custom services. The frontend needs be able to fetch a (paginated, sorted) list of ALL services for an enrollment|project that includes both HUD- and Custom services.

I'm not sure how to make that happen if we are only storing HUD Services in the `Services` table and only storing custom services in the `CustomServices` table. I don't think it's possible in rails to do what we need (basically 1 AR model that is backed by both tables).

@eanders would love to get your thoughts on this next week.

some potential paths:
* Change our mind and store all custom services in the Services table with 999s.
    * Note: this is different from the (CE) Assessment / CustomAssessment distinction because we never actually want/need to resolve CustomAssessments and (CE) Assessments as a single collection. I think there is probably a case here for using the hud table.
* Duplicate each HUD service rendered in both tables (Services and CustomServices) and have the API resolve CustomServices only
* Some Rails magic I don't know about 🤞

Reply:
Let's discuss on Monday. Maybe a view with a computed STI column. Alternatively, a simple table with the id and type of the service parent.

Reply:
Sounds good, thanks. Those both sound like reasonable options.